.gitignore

```diff
@@ -109,3 +109,15 @@ vgcore*
 /nanomsg-1.0.0/
 /nnpy/
 /thrift-0.11.0/
+
+# vscode configuration
+.vscode/
+
+# idea configuration
+.idea/
+
+# files generated by checkinstall
```

Comment:
> doc-pak/
backup-*
description-pak
*.deb 

Are they all generated by check install?

Reply:
I'll rerun the checkinstall to check this

Reply:
After running the checkinstall, git told me that there are following newly created files:

- backup-072320221847-pre-runtime-programmable.tgz
- description-pak
- doc-pak/
- runtime-programmable_1.0.0-1_amd64.deb

Under doc-pak/ there are:
```
.
|-- docs
|   |-- JSON_format.md
|   |-- p4dbg_user_guide.md
|   |-- performance.md
|   |-- runtime_CLI.md
|   `-- simple_switch.md
|-- LICENSE
|-- README.md
`-- VERSION

```